# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../examine/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/examine_client.c.vanilla	2004-06-21 22:08:42.991214496 +0200
+++ ./src/examine_client.c	2004-06-21 22:09:21.631340304 +0200
@@ -25,6 +25,9 @@
 Ecore_Config_Ipc_Server_List *examine_client_server;
 examine_prop   *prop_list;
 
+void examine_client_list_props_cb(void);
+void examine_client_get_val_cb(void);
+
 /*****************************************************************************/
 
 
